lib/src/modules/mod.rs

```diff
@@ -225,6 +231,7 @@ pub mod mods {
     /// dispatch version of the structure returned by the YARA module.
     pub fn invoke_dyn<T: protobuf::MessageFull>(
         data: &[u8],
+        meta: Option<&[u8]>,
```

Comment:
We have `invoke` and `invoke_dyn`, and now we are adding `invoke_with_meta`. For consistency we should have `invoke_with_meta_dyn` instead of changing the signature for `invoke_dyn`.

Reply:
done in [cca5adf](https://github.com/VirusTotal/yara-x/pull/183/commits/cca5adfaeb3671482a47f5b0a1a8e3f7d2a70fe8)